app/models/tweet.rb

```diff
@@ -1,4 +1,8 @@
 class Tweet < ApplicationRecord
   belongs_to :user
+  delegate :username, to: :user
+
   validates :text, presence: true, length: { maximum: 140 }
+
+  scope :latest, -> { order(:created_at) }
```

Comment:
Kung `latest`, maybe this is more apt:
`scope :latest, -> { order(created_at: :desc) }`

Reply:
Forgot about the `:desc`, thanks po for catching that one. :smile: